funflow/funflow.cabal

```diff
@@ -29,8 +29,6 @@ Library
                    , Control.Arrow.Free
                    , Control.Funflow
                    , Control.Funflow.Cache.TH
-                   , Control.Funflow.ContentHashable
```

Comment:
Do we want to keep these modules (as reexports) for backward-compatibility? That way the change is totally transparent for downstream users

(otoh I'm not sure whether this modules are used a lot in client code)

Reply:
I don't think they are, that's why I chose not to forward them.